docs/users/invitations.mdx

```diff
@@ -10,7 +10,14 @@ Once you create an invitation, Clerk sends an email to the invited user with a u
 Invitations expire after a month. If the user clicks on an expired invitation, they will get redirected to the application's sign-up page and will have to go through the normal sign-up flow. Their email address will not be auto-verified.
 
 > [!TIP]
-> Invitations are only used to invite users to your application. The application will still be available to everyone even without an invitation. If you're looking to create an invitation-only application, please refer to Clerk's [restrictions](/docs/authentication/configuration/restrictions) options.
+> Invitations are only used to invite users to your application. The application will still be available to everyone even without an invitation. If you're looking to restrict access to invited users only, please refer to Clerk's [Restricted sign-up mode](/docs/authentication/configuration/sign-up-modes#restricted).
+
+## Using Clerk Dashboard
+
+To invite users, navigate to the **Users** page from the top-level menu, then select the **Invitations** tab. Select **Invite user** and enter the email address of the person you want to invite. Optionally, set an expiration date for the invitation. Once you've entered the necessary details, click **Send invite** to send the invitation email.
+
+> [!TIP]
+> The same page will be available throught the top-level menu, when you are in the **Restricted** sign-up mode.
```

Comment:
This needs to be moved under the "## Create an invitation" section. That section will need to be rewritten, because as it stands, it says invitations can only be created via the Backend API. But we clearly support it in the Dashboard. So we need to rewrite this section

Reply:
Updated the section for both create and revoke to have both options. Also updated to suggest restricted sign-up mode instead of allowlist.